python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+from triton.experimental.gluon.language import _core as ttgl
+from triton.experimental.gluon.language._core import builtin
+from triton._C.libtriton import ir
+
+if TYPE_CHECKING:
+    from ..._semantic import GluonSemantic
+
+__all__ = ["buffer_load", "buffer_store"]
+
+
+@builtin
+def buffer_load(ptr, offsets, mask=None, other=None, stride=None, cache=None, _semantic=None):
+    """
+      This op is lowered into BufferLoadOp of TritonAMDGPU directly.
```

Comment:
When we write documentation, we need to be aware of who is the target audience to make sure the documentation makes sense for them. These ops are meant for kernel developers who may not be familiar with compiler internal lowering process. So the way it is written as of now is providing no much value to them. Can you refer to the op definition in the `TritonAMDGPUOps.td` to properly define the semantics here? You can largely copy the text there like what https://github.com/triton-lang/triton/pull/7746 did.